project/src/components/error-message/error-message.tsx

```diff
@@ -0,0 +1,27 @@
+import { useAppSelector } from '../../hooks';
+
+function ErrorMessage(): JSX.Element | null {
+  const{error} = useAppSelector((state) => state);
+
+  if(error) {
+    return (
+      <div
```

Comment:
```suggestion
  if(!error) {
    return null
```
лучше вот так)
более понятней